MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -48,11 +48,26 @@
 @section Scripts {
     @Scripts.Render("~/bundles/jqueryval")
     <script>
-        $(".js-login-jump-icon").click(function () {
+        $(".js-login-click-to-scroll").click({ param: 500 }, scrollLogin); // "{param: "data"}" can be used to pass parameter as an object
+
+        if ($(".js-invalid-login-message").length > 0) {
+            scrollLogin()
+        } else {
+            // Executes scrollLogin(speed) in x mileseconds
+            setTimeout(function () { scrollLogin(2000) }, 1500);
+        }
+
+        // Scroll down to login panel
+        function scrollLogin(speed) {
```

Comment:
there are many approaches to initialize a value/param.
in this situation, function scrollLogin(speed = 0) would be good enough. 

we don't use if(speed == undefined), usually we can say
if(!speed) 
or if(typeof speed === "undefined")
or in some situation speed = speed || 100